buildSrc/src/main/groovy/org/elasticsearch/gradle/test/VagrantFixture.groovy

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.gradle.test
+
+import org.elasticsearch.gradle.vagrant.VagrantCommandTask
+import org.gradle.api.Task
+
+/**
+ * A fixture for integration tests which runs in a virtual machine launched by Vagrant.
+ */
+class VagrantFixture extends VagrantCommandTask implements Fixture {
+
+    private Task stopTask = null
+
+    public VagrantFixture() {
+        project.afterEvaluate {
+            // force the initialization of the stop task at the end
```

Comment:
I'm running into a strange problem with this. The build and tests run fine but the stop task on the vagrant fixture ceases to pick up the environment variables for the vagrant command, and thus cannot find the original Vagrantfile used to set up the krb5kdc vm. Without it, the command exits complaining that it doesn't know the vm that it's being asked to shut down. 

I'm not sure I'm able to spot where exactly it's dropping the variables on the floor, since all my attempts at logging the values have returned exactly what I expected them to.